src/test/java/com/example/training/repository/SimpleListRepositoryTest.java

```diff
@@ -24,4 +24,17 @@ public void When_GettingUserById_Expect_CorrectResult() {
     }
 
     //ToDo implement unit tests for SimpleListRepository
+    @Test
+    public void When_AddingUser_Expect_CorrectResult() {
+        REPOSITORY.addUser(NEW_TEST_USER);
+        assertThat(REPOSITORY.getAllUsers().size(), is(5));
+        // rollback new user
```

Comment:
delete all comments. It's clear when you read this code.
Use comments only for some bug work-arounds or some specific algorithms.